css/components/_btn.scss

```diff
@@ -0,0 +1,77 @@
+
+.btn {
+  border: 1px solid transparent;
+  border-radius: .25rem;
```

Comment:
`rem` aren't a good unit for things like borders. Use `em` instead. More info at: https://css-tricks.com/rems-ems/